app/models/education_benefits_claim.rb

```diff
@@ -3,9 +3,10 @@
 require 'attr_encrypted'
 class EducationBenefitsClaim < ActiveRecord::Base
   FORM_TYPES = %w[1990 1995 1990e 5490 5495 1990n 0993 0994].freeze
-  # TODO: TO BE REMOVED IN NEXT PR WHEN REPORTING CAN INCLUDE 0994
-  FORM_TYPES_MINUS_0994 = %w[1990 1995 1990e 5490 5495 1990n 0993].freeze
 
+  # 'vettec' type MUST be the last value in the array. If it isn't the last value,
+  # then it would impact the EducationForm::CreateDailyFiscalYearToDateReport and CreateDailyYearToDateReport
```

Comment:
I think if you wrap class names in brackets like so:

```suggestion
  # then it would impact the {EducationForm::CreateDailyFiscalYearToDateReport} and {CreateDailyYearToDateReport}
```

then YARD will auto-link to them

Reply:
wrapped the class name into bracket